### DISCUSSION TITLE
Change displayed date format

### DISCUSSION THREAD

**Discussion by K1ngJony** [upptime/upptime#589] [Q&A]
Is it possible to change the date format either to something less ambiguous, like June 06 2022, or even ISO 8601? I can't find a configuration for it.
All of my past incidents and scheduled maintenance statuses show US date format. As I'm in Australia this gets very confusing. (My currently scheduled maintenance is due to start tomorrow, June 11, but anyone looking at it form here would think it's scheduled for November 6)

**Comment by AnandChowdhary** [comment]
I think we use `toLocaleString()` everywhere (e.g., https://github.com/upptime/status-page/blob/master/src/components/Incident.svelte#L98), so the dates should be formatted based on your system, so if you have US English set as your computer's language, that's what you should see.

I think you're right, we can totally add a configuration property for the default locale. Happy to accept a PR!
